numpy_cpp-custom-matrix/test.py

```diff
@@ -26,3 +26,12 @@
 print("A = ", A)
 print("B = ", B)
 print("A * B = ", example.Multiply(A,B,C,2,2,3))
+
+print()
+A = np.array([[1,1,1],[2,2,2]])
```

Comment:
Try to use a more distinctive example: instead of repeating 1 in the first row and 2 in the second row, use this: A = np.array([[1,2,3],[4,5,6]])